media/js/firefox/hello/start.js

```diff
@@ -13,6 +13,9 @@
     var highlightTimeout;
     var highlightsSupressed = false;
     var tourStep = 'default';
+    var tourSource = getParameterByName('utm_content').replace('hello-tour_OpenPanel_','');
+    // Help readability for the user
+    replaceURLState('utm_content=hello-tour_OpenPanel_' + tourSource, 'utm_content=' + tourSource);
```

Comment:
If this is just for readability, I'd say maybe this is a bit superfluous - the param is right at the end of a string of utm parameters anyhow. I don't think people will really see it either way.